mantle/platform/api/do/api.go

```diff
@@ -139,7 +139,7 @@ func (a *API) CreateDroplet(ctx context.Context, name string, sshKeyID int, user
 			Size:              a.opts.Size,
 			Image:             a.image,
 			SSHKeys:           []godo.DropletCreateSSHKey{{ID: sshKeyID}},
-			IPv6:              true,
+			IPv6:              false,
```

Comment:
Is it possible to tell whether the specified image is a custom image?  I'd be tempted to conditionalize this.  One day hopefully we will support running from a standard image, in which case we should probably keep this enabled.

(If it's a substantial amount of work, I'd say skip it for now, and put a comment here.)

Reply:
We specify the image by name, which might not be unique. However, I think that in most cases we could query for private images:
```
curl -X GET \
  -H "Content-Type: application/json" \
  -H "Authorization: Bearer $DIGITALOCEAN_TOKEN" \
  "https://api.digitalocean.com/v2/images?private=true"
```

which gives us:
```
{"images":[{"id":94981113,"name":"fcos","distribution":"Fedora","slug":null,"public":false,"regions":["nyc3","sfo2"],"created_at":"2021-11-03T18:53:26Z","min_disk_size":15,"type":"custom","size_gigabytes":0.8,"description":"","tags":["mantle"],"status":"available","error_message":""}],"links":{},"meta":{"total":1}}%
```

Where we could make sure that if an object with the name we passed matches the name + type=custom then we set the flag to false.

That sounds like a good approach to you?

Reply:
Works for me.  We probably shouldn't do the query in every `CreateDroplet` call though.  `CreateDroplet` can take a flag argument, and the DigitalOcean `Flight` can call into `platform/api/do` to do the query once at the beginning of the test run.